lib/Server.js

```diff
@@ -702,7 +699,10 @@ class Server {
       return true;
     }
 
-    if (!headerToCheck) headerToCheck = 'host';
+    if (!headerToCheck) {
+      headerToCheck = 'host';
+    }
```

Comment:
Add `{` and `}` to all construction (better reading), we should enforce this rule in eslint, let's do it in other PR